webbpsf/opds.py

```diff
@@ -261,7 +261,7 @@ def powerspectrum(self, max_cycles=50, sampling=5, vmax=100, iterate=False):
 
         """
 
-        # import SFT
+        import SFT
```

Comment:
This edit is incorrect; the commented-out import statement should just be deleted entirely. (SFT.py hasn't existed in this repo for many many years; that evolved into poppy's matrix_dft.py)

Reply:
@mperrin `def tvcircle` uses SFT.  Should that entire method be removed?

Reply:
There are clearly some functions here which haven't been used or called in forever, in particular the `powerspectrum`function . Not included in the test suite, since clearly it doesn't work given that it's calling a file that doesn't exist.